code/game/machinery/doors/shutters.dm

```diff
@@ -19,15 +19,15 @@
 	add_fingerprint(user)
 	if(istype(I,/obj/item/tool/crowbar/halligan))
 		var/obj/item/tool/crowbar/halligan/H = I
-		if(density) //can't cut an open door open, dumbass
+		if(density) //can't cut an open door open
+			if(pried_open) //shutter is already fully cut open
```

Comment:
there is no reason for this to be its own if statement instead of density && !pried_open